### PR TITLE
Fix cfmon

### LINKED ISSUE #170: CFmon variable clisccp and clcalipso: won't convert
Note that other CFmon variables works fine, the failed two variables have additional dimensions.  

clisccp | ISCCP Cloud Area Percentage | FISCCP1_COSP | clisccp = FISCCP1_COSP | %(time, plev7c, tau, lat, lon)
-- | -- | -- | -- | --

clcalipso | CALIPSO Percentage Cloud Cover | CLD_CAL | clcalipso = CLD_CAL | %(time, alt40, lat, lon)
-- | -- | -- | -- | --

### PR DESCRIPTION
**Pull request by chengzhuzhang**
Closes #170 
fix clisccp and clcalipso handlers.

- clisccp: convert pressure units only if the original units is hPa (as those in E3SM v1 data) for pressure coordinate `cosp_prs`. 
- clcalipso: use numpy array for coordinate values.

### CODE REVIEW

File: e3sm_to_cmip/cmor_handlers/clcalipso.py [hunk 1]
```diff
@@ -24,13 +24,13 @@ def write_data(varid, data, timeval, timebnds, index, **kwargs):
     clcalipso = CLD_CAL with alt40 levels
     """
     if kwargs.get('simple'):
-        return data[ RAW_VARIABLES[0] ][index, :]
+        return data[ RAW_VARIABLES[0] ][index, :].values
```
Comment (tomvothecoder) [inline e3sm_to_cmip/cmor_handlers/clcalipso.py:27]:
```suggestion
        return data[RAW_VARIABLES[0]][index, :].values
```

File: e3sm_to_cmip/cmor_handlers/clcalipso.py [hunk 2]
```diff
@@ -24,13 +24,13 @@ def write_data(varid, data, timeval, timebnds, index, **kwargs):
...
     cmor.write(
         varid,
-        data[RAW_VARIABLES[0]][index, :],
+        data[RAW_VARIABLES[0]][index, :].values,
         time_vals=timeval,
         time_bnds=timebnds)
-    return data[ RAW_VARIABLES[0] ][index, :]
+    return data[ RAW_VARIABLES[0] ][index, :].values
```
Comment (tomvothecoder) [inline e3sm_to_cmip/cmor_handlers/clcalipso.py:33]:
```suggestion
    return data[RAW_VARIABLES[0]][index, :].values
```

**Review by tomvothecoder — APPROVED** [review]
Minor code formatting suggestions. Otherwise, looks good.

**Comment by chengzhuzhang** [comment]
Thank you @tomvothecoder

**Pull request merged**